src/Akka.Streams.Kafka/Settings/Subscriptions.cs

```diff
@@ -24,20 +26,36 @@ public interface IManualSubscription : ISubscription { }
     public interface IAutoSubscription : ISubscription
     {
         /// <summary>
-        /// Partition events handler
+        /// Optional. Partition events handler.
         /// </summary>
         Option<IPartitionEventHandler> PartitionEventsHandler { get; }
         
+        /// <summary>
+        /// Optional actor that receives rebalance events as messages.
+        /// </summary>
+        Option<IActorRef> RebalanceListener { get; }
```

Comment:
The `RebalanceListener` is part of the `IAutoSubscription` and this gets called from inside the `AsyncCallbacks: IPartitionEventHandler`